src/expr/scalar/func.rs

```diff
@@ -1329,6 +1329,18 @@ fn date_trunc<'a>(a: Datum<'a>, b: Datum<'a>) -> Result<Datum<'a>, EvalError> {
     }
 }
 
+fn date_trunc_timestamptz<'a>(a: Datum<'a>, b: Datum<'a>) -> Result<Datum<'a>, EvalError> {
```

Comment:
add a comment explaining that this works because we store timestamptz's as utc timestamps under the hood